src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java

```diff
@@ -1529,6 +1526,7 @@ public List<Tuple3<HighlightSeverity,TextRange,LSPCodeActionFix>> getSilentAnnot
     public void refreshAnnotations() {
         if (!annotationsRefreshed) {
             updateErrorAnnotations();
+            silentAnnotations.clear();
```

Comment:
When the position of a specific code that has code actions available changes, the silent annotation related to the previous position will still remain in the list and display code actions in the old position. Therefore clear the list after showing the code actions.